creme/geolocation/populate.py

```diff
@@ -51,6 +56,7 @@ def populate(self):
         )
 
         SettingValue.objects.set_4_key(setting_keys.google_api_key, '')
+        SettingValue.objects.set_4_key(setting_keys.use_entity_icon_key, False)
```

Comment:
- The second line is now useless.
- Improve the first line (`get_or_create()` + move before `already_populated`)